pageserver/src/timeline_metrics.rs

```diff
@@ -0,0 +1,306 @@
+use metrics::core::{AtomicU64, GenericCounter, GenericGauge};
```

Comment:
I'd suggest renaming the file to just `metrics.rs`. Some of the metrics are system-wide, not per-timeline.

Reply:
Agreed, started to look much more generic soon after the start.

Reply:
Leads to naming conflict though. https://github.com/neondatabase/neon/blob/07b4ace52fd6097e74d982ba4dbd74dd28a4f8dc/pageserver/src/lib.rs#L29 gets ambiguous. Although I guess I could move the rest of the definitions to the new metrics to avoid that. What do you think?

Reply:
Maybe `pageserver_metrics.rs` then.

Reply:
I just moved the rest of the metrics to `metrics.rs` as there weren't that many. It would probably eliminate the need to use https://docs.rs/metrics/latest/metrics/# outside of `src/metrics.rs`. Would also look more like in safekeeper.

Reply:
@hlinnaka what do you think about the current solution? Kept the metrics in `metrics.rs` and moved the rest of the definitions there, so it eliminates the need for metric conflict.

Reply:
Works for me, thanks